tests/e2e/framework/kubernetes.go

```diff
@@ -725,6 +755,31 @@ func (k *KubeInfo) deepCopy(src map[string][]string) map[string][]string {
 	return newMap
 }
 
+func (k *KubeInfo) fileCopy(src string, dst string) error {
```

Comment:
Could be replaced by `./tests/util/CopyFile`?

Reply:
thanks used that function instead.